controllers/api.articles.comments.controller.js

```diff
@@ -45,3 +47,20 @@ exports.postComment = (req, res, next) => {
     })
   );
 };
+
+exports.deleteComment = (req, res, next) => {
+  const comment_id = req.params.comment_id;
+  if (isNaN(comment_id)) {
+    return next({
+      status: 400,
+      msg: "Error - invalid comment_id",
+    });
+  }
```

Comment:
This doesnt need to be done manually as psql should throw an error for this. You should also not have error handling in the controller, but instead in the model instead.